UPGRADING

```diff
@@ -45,7 +45,7 @@ PHP 8.2 UPGRADE NOTES
     stripos, strripos, lcfirst, ucfirst, ucwords, str_ireplace,
     array_change_key_case and sorting with SORT_FLAG_CASE use ASCII case
     conversion.
-  . str_split no longer returns an empty array on empty string.
+  . str_split returns an empty array on empty string.
```

Comment:
We might also mention the previous behavior

Reply:
I did something. let me know what you think. Again, technical documentation is maybe not my strong point :)